cdragontoolbox/__main__.py

```diff
@@ -429,6 +442,14 @@ def create_parser():
     subparser.add_argument('bin',
                            help="BIN file to extract")
 
+    # skn files commands
+
+    subparser = subparsers.add_parser('skn-extract',
+                                      help="Extract an SKN file to a folder")
+    subparser.add_argument('skn',
+                           help="SKN file to extract")
+    subparser.add_argument('output',
+                           help="output folder")
```

Comment:
I think it would be better to use a `-o`/`--output` option, as elsewhere.
You could make it default (if `None`/unset) to a directory based on `skn` value.

Also, `folder` -> `directory`.

Reply:
If you're fine with a (unspecified) default value, I'll gladly add that. This will help my personal use case as well.

Reply:
Yes, it's fine. Juste use `None` as default value and convert it to the final value later, based on `args.skn`.